package.json

```diff
@@ -11,5 +11,8 @@
     "lint": "cd frontend && npm run lint",
     "postinstall": "npm run build && npm run deploy",
     "start": "cd frontend && npm start"
+  },
+  "dependencies": {
+    "universal-cookie": "^4.0.2"
```

Comment:
Not sure how this got here, but it shouldn't be. And you'll want to reset changes to the root package-lock.json.